client/payment-details/summary/test/index.test.tsx

```diff
@@ -427,6 +427,13 @@ describe( 'PaymentDetailsSummary', () => {
 		screen.getByRole( 'button', {
 			name: /Accept dispute/,
 		} );
+
+		// Refund menu is not rendered
+		expect(
+			screen.queryByRole( 'button', {
+				name: /Translation actions/i,
```

Comment:
Note that the label for the refund menu button is `Translation actions` – I've opened #7964 to correct this label.